usaspending_api/database_scripts/matviews/functions_and_enums.sql

```diff
@@ -1,8 +1,5 @@
 -- Postgres extensions, functions, and enums necessary for some views
 
-CREATE EXTENSION IF NOT EXISTS intarray;
-
-
```

Comment:
Did you intend to remove this? Are you confirming that no other SQL code somehow needed this postgres extension? Or maybe it's native now in our version of Postgres?

Reply:
It's included in the migration for `subaward_search` as its needed for the vectors.